client/src/actions/appointment.js

```diff
@@ -13,16 +14,37 @@ export const updateMentor = value => {
   history.push("/schedule");
   return {
     type: MENTOR,
-    payload: value
+    payload: value.mentor
   };
 };
 
 export const fetchAvailibilites = mentor => {
   /* fetch data for specific mentor*/
+  return dispatch => {
+    axios
+    .get(`/getavailabilities?mentor=${mentor}`)
+    .then(res => {
+      dispatch({
+        type: AVAILIBILTY,
+        payload: res.data
+      })
+    })
+  };
+};
 
-  return {
-    type: AVAILIBILTY,
-    payload: ["1", "2", "3"] //dummy
+export const fetchMentors = () => {
+
+  return dispatch => {
+    axios
+      .get("/getmentordata")
+      .then(res => {
+        console.log(res.data);
```

Comment:
Uneeded console log here - might be worth doing a find all for console log before you push up?